Update toolbox.py

ZhenluRen opened the pull request:
For dd:
		for j in range(ne_temp):
			d_BM = d_AB
			d_BM_max = (ne_temp - 3) * d_AB
			ne_temp = ne_temp - 1
			while d_BM <= d_BM_max:
For slm:
		for j in range(ne_temp):
			d_MN = d_AM
			d_MN_max = (ne_temp - 3) * d_AM
			ne_temp = ne_temp - 1
			while d_MN <= d_MN_max:

ZhenluRen committed:
Update toolbox.py

For dd:
		for j in range(ne_temp):
			d_BM = d_AB
			d_BM_max = (ne_temp - 3) * d_AB
			ne_temp = ne_temp - 1
			while d_BM <= d_BM_max:
For slm:
		for j in range(ne_temp):
			d_MN = d_AM
			d_MN_max = (ne_temp - 3) * d_AM
			ne_temp = ne_temp - 1
			while d_MN <= d_MN_max:

jc-olalla reviewed (commented):
I don't understand the change.

ZhenluRen commented:
The previous scripts didn't use the last electrode to generate the sequence, then I tried to make this change, then the sequence file can include the number of the last electrode.

ZhenluRen committed:
Update toolbox.py

add function "pseudo_section_with_depth"
add function "dd_generation"
add loop "remove overlap configurations"
